### PR TITLE
fix(homepage): fixed-mat-icons

### PR DESCRIPTION
**Pull request by abhishek-mishra019**
Signed-off-by: Abhishek Mishra [am1426620@gmail.com](mailto:am1426620@gmail.com)
### Functionality:
to fix mat-icons in project

### Solution:
fixed the mat icons and also made the header, footer redundant in each page

### Risk level:
- [x] high 
- [ ] medium
- [ ] low

### How to test:
clone the repo and go live

### CODE REVIEW

File: header/header.html [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
             <nav id="navbar" class="navbar">
-                <a class="navbar-brand" href="#">
+                <a class="navbar-brand" href="index.html">
                     <img id="brand_img" src="assets/worktezName.png" alt="Worktez" width="125px" height="45px">
                 </a>
                 <ul class="nav nav-pills" style="display: inline-flex; font-size: 12px;">
                     <li class="nav-item" >
-                        <a class="nav-link" href="pricing/pricing" style="color: #838CAB;">Pricing</a>
+                        <a class="nav-link" href="/pricing/pricing.html" style="color: #838CAB;">Pricing</a>
                     </li>
```
Comment (sanjaykrishna1203) [inline header/header.html]: The .html was purposely removed to remove the .HTML extension in the URL.
Please follow the same.

File: pricing/pricing.html [hunk 2]
```diff
@@ -34,9 +34,46 @@
...
     <div class="app-theme">
 <!-- Header -->
-<div id="header"></div>
+<div class="sHeader pb-2">
+  <div class="row header" style="background-color: #ffffff; color: #1B1F23; box-shadow: 1px 1px 4px 1px #ebecefa5;">
+      <div class="col-md-2 col-1"></div>
+      <div class="col">
+          <nav id="navbar" class="navbar">
+              <a class="navbar-brand" href="/index.html">
+                  <img id="brand_img" src="/assets/worktezName.png" alt="Worktez" width="125px" height="45px">
```
Comment (sanjaykrishna1203) [inline pricing/pricing.html:43]: Same here

File: pricing/pricing.html [hunk 3]
```diff
@@ -34,9 +34,46 @@
...
+              </a>
+              <ul class="nav nav-pills" style="display: inline-flex; font-size: 12px;">
+                  <li class="nav-item" >
+                      <a class="nav-link" href="/pricing/pricing.html" style="color: #838CAB;">Pricing</a>
+                  </li>
```
Comment (sanjaykrishna1203) [inline pricing/pricing.html]: ..

File: pricing/pricing.html [hunk 4]
```diff
@@ -169,7 +206,76 @@
...
+              <div class="col-md-3 col" (click)='termsandcondition()' id="txtCenter">Terms & Conditions</div>
+              <div class="col-md-1 col" id="txtCenter">|</div>
+              <div class="col-md-2 col" (click)='privacy()'id="txtCenter">Privacy Policy</div>
+              <div class="col-md-1 col" id="txtCenter">|</div>
+              <div class="col-md-2 col" (click)="copyright()"id="txtCenter">License</div>
+              <div class="col-md-3 col text-right"id="txtCenter" style="text-align: right;">
+                  <img class="arrow" src="../assets/material-icons/copyright_black_24dp.svg" alt="">
+                      Copyright Worktez 2022. All Rights Reserved</div>
+          </div>
+      </div>
```
Comment (sanjaykrishna1203) [inline pricing/pricing.html]: 2022 -> 2023

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

**Review by sanjaykrishna1203 — APPROVED** [review]

File: pricing/pricing.html [hunk 5]
```diff
@@ -16,7 +16,7 @@
  <html lang="en">
  <head>
 
-    <link rel="stylesheet" href="..\pricing\pricing.css">
+    <link rel="stylesheet" href="../pricing/pricing.css">
```
Comment (sanjaykrishna1203) [inline pricing/pricing.html:19]: Not sure with the path.. 
Please check for its working after deployment

Comment (abhishek-mishra019) [reply pricing/pricing.html:19]: sure

**Pull request merged**